modules/instruments/test/TestPlan.md

```diff
@@ -43,3 +43,14 @@ instrument for that candidate. Enter a date (same a step 4) posterior to the DoD
 12. Verify Validity flags are functioning for instruments.
     - **Validity:** can be marked as “Valid”, “Questionable”, or “Invalid”. Whether or not this flag is shown for an instrument is by the boolean $ValidityEnabled. Whether the field is required before flagging an instrument as complete is determined by $ValidityRequired.
     - You can test the forgoing flags by switching the corresponding boolean of the test instruments to either `true` or `false`.
+13. Dashboard Widget - "My Tasks" for Incomplete forms
+    - Ensure the total of Incomplete forms correspond with the correct 
+        number of candidates inside the statistics/statistics_site module.
+    - Click on the total number of Incomplete forms and check if redirection
+        to the statistics/statistics_site module succeeds.
+14. Candidate Widget - "Behavioural Data" for Candidate instruments entry.
+    - Ensure the appropriate visit labels are visible for the candidate. 
+    - The visit status, subproject status, site, date of visit, and age should be shown.
+    - Clicking on a visit should redirect to the instrument_list module for the candidate.
+    - Test making a new Time Point for a candidate and assign a Visit label. The Behavioural Data widget should update 
+        to include the new Time Point by showing the visit label with the subproject, site, and status as not started.
```

Comment:
```suggestion
        to include the new Time Point by showing the instrument with the subproject, site, and status as not started.
```

Reply:
I made the 2 previous comment with a length of time in between. This line is now superfluous.